packages/core/src/amazonq/explorer/amazonQChildrenNodes.ts

```diff
@@ -25,12 +27,21 @@ export const qExtensionPageCommand = Commands.declare('aws.toolkit.amazonq.exten
     void vscode.env.openExternal(vscode.Uri.parse(`vscode:extension/${VSCODE_EXTENSION_ID.amazonq}`))
 })
 
-export const dismissQTree = Commands.declare('aws.toolkit.amazonq.dismiss', () => async () => {
-    await globals.context.globalState.update(amazonQDismissedKey, true)
-    await vscode.commands.executeCommand('setContext', amazonQDismissedKey, true)
-})
+export const dismissQTree = Commands.declare(
+    { id: '_aws.toolkit.amazonq.dismiss', compositeKey: { 1: 'source' } },
```

Comment:
Update: I see you used it correctly in the other uses.

---

The `1` in `{ 1: 'source'}` means the arg at index `1` of the below `(source: string`, which is incorrect. I'm assuming that the source is not properly set in the final metric.

You can verify in the Toolkit Logs, once you execute this command search for something like `command: _aws.toolkit.amazonq.dismiss` in a `vscode_executeCommand` metric. Then see if the `source` field is what you expect.

Reply:
missed this one, fixed